### DISCUSSION TITLE
Suggestion: Add Restart as Admin

### DISCUSSION THREAD

**Discussion by BlackOfWorld** [microsoft/terminal#14361] [Ideas]
Currently, I have to go launch it from windows search, which takes some time and can be very annoying.  Having a "Restart as Admin" button would remove that annoyance and would be amazing.

**Comment by gurjui** [comment]
I have the same issue, but I solved it with a simple function I added to the profile: https://gist.github.com/gurjui/c2250774cda2641077bd1d8e10c8756c
Not ideal but it works

**Reply by BlackOfWorld** [reply]
> I have the same issue, but I solved it with a simple function I added to the profile: https://gist.github.com/gurjui/c2250774cda2641077bd1d8e10c8756c Not ideal but it works

I'd recommend using https://github.com/gerardog/gsudo, but it does not solve the issue, every time you create a new tab it will not be run as admin
